lms/static/sass/_build-discussion-editor.scss

```diff
@@ -0,0 +1,4 @@
+// From _build-course.scss
+
+@import "course/discussion/form-wmd-toolbar";
```

Comment:
@andy-armstrong Yes, that works. See https://github.com/edx/edx-platform/pull/13483/commits/e811df70457d6ece87f51d76319fd89cbce6c456.